bot/exts/utils/snekbox.py

```diff
@@ -36,6 +36,39 @@
     re.DOTALL                               # "." also matches newlines
 )
 
+TIMEIT_SETUP_WRAPPER = """
+import atexit
+import sys
+from collections import deque
+
+if not hasattr(sys, "_setup_finished"):
+    class Writer(deque):
+        def __init__(self):
+            super().__init__(maxlen=1)
+
+        def write(self, string):
+            if string.strip():
+                self.append(string)
+
+        def read(self):
+            return self.pop()
+
+        def flush(self):
+            pass
+
+    sys.stdout = Writer()
+
+    def print_last_line():
+        if sys.stdout:
+            print(sys.stdout.read(), file=sys.__stdout__)
+
+    atexit.register(print_last_line)
+    sys._setup_finished = None
+{setup}
+"""
```

Comment:
This should be documented more

Reply:
I agree wholeheartedly. This code sorely needs block comments and docstrings just like code that isn't inside of a triple quoted string would. Just because the linter doesn't mind, doesn't mean we should lower our readability standards.

Reply:
Agreed. Resolved in https://github.com/python-discord/bot/pull/1602/commits/afdb2b4d5ebeda20d90e794711ca7e56244cfd6b.